charts/sn-platform/templates/bookkeeper/bookkeeper-cluster.yaml

```diff
@@ -28,6 +28,9 @@ metadata:
   labels:
     {{- include "pulsar.standardLabels" . | nindent 4 }}
     component: {{ .Values.bookkeeper.component }}
+    {{- with .Values.bookkeeper.labels }}
+    {{ toYaml . | indent 4 }}
```

Comment:
```suggestion
{{ toYaml . | indent 4 }}
```

Reply:
same for other places where we include yaml

Reply:
Thanks, fixed